content/en/logs/guide/logs-rbac.md

```diff
@@ -123,7 +119,7 @@ This section explains how to create two roles, `ACME Admin` and `ACME User`; how
 {{< tabs >}}
 {{% tab "UI" %}}
 
-In the [Team Section][1] of Datadog, use the Add Role button within the Role tab to create the new `ACME Admin` and `ACME User`roles.
+In the [Groups Section][1] of Datadog, use the Add Role button within the Role tab to create the new `ACME Admin` and `ACME User`roles.
```

Comment:
See: https://app.datadoghq.com/organization-settings/roles

In the left-hand sidebar, `Roles` falls under a section called `Groups`.